tests/utils.py

```diff
@@ -0,0 +1,49 @@
+import pennylane as qml
+import pennylane.numpy as pnp
+
+from maskit.masks import FreezableMaskedCircuit, MaskedCircuit
+
```

Comment:
Is there a reason why these functions are all named as ``_protected``? Since they are used in other modules, I would name them ``public``.

Reply:
No specific reason just a number of refactorings 😸